src/thirdparty/libtelnet/libtelnet.c

```diff
@@ -1667,3 +1667,12 @@ void telnet_begin_zmp(telnet_t *telnet, const char *cmd) {
 void telnet_zmp_arg(telnet_t *telnet, const char* arg) {
 	telnet_send(telnet, arg, strlen(arg) + 1);
 }
+
+/* send CHARSET OF UTF-8 command */
+void telnet_send_utf8(telnet_t *telnet) {
```

Comment:
this function belongs either in src/net/telnet.cc or src/net/telnet-ext.cc  ,  it shouldn't be in the libtelnet.c